.github/workflows/main.yml

```diff
@@ -0,0 +1,50 @@
+name: Main workflow
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+ eslint:
+   name: ESLint
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code  
+        uses: actions/checkout@v3
+      - name: Install modules
+        run: yarn
+      - name: Run ESLint
+        run: yarn run eslint . --ext .js,.jsx,.ts,.tsx
+ test:
+   name: Test
+   runs-on: ubuntu-latest
+   needs: eslint
+   steps:
+      - uses: actions/checkout@v2
+      - uses: borales/actions-yarn@v2.3.0
+        with:
+          cmd: install
+      - uses: borales/actions-yarn@v2.3.0
+        with:
+          cmd: test
```

Comment:
why cmd was used here, not `run: yarn test`?

Reply:
changed to yarn